cloudbuild.yaml

```diff
@@ -46,16 +46,6 @@ steps:
     args:
       - build
     id: build
-  # Run the built images for 5 seconds to make sure that the entrypoint and default config works out of the box
-  - name: gcr.io/cloud-builders/docker
-    dir: ./build
-    entrypoint: bash
-    args:
-      - '-c'
-      - 'timeout --signal=INT --preserve-status 5s docker run --rm ${_REPOSITORY}quilkin:$(make version)'
```

Comment:
Rather than deleting, could make it (something like)

`'timeout --signal=INT --preserve-status 5s docker run --rm ${_REPOSITORY}quilkin:$(make version)' run`

To make it run in `run` mode. 

Although one could argue the integration tests also test this (although this would provide a nicer error message if something basic was broken). I could go either way on this one.